lib/tasks/manage.rake

```diff
@@ -108,7 +108,7 @@ namespace :manage do
       if active_subscriptions.empty?
         puts "Subscriber #{email_address} already unsubscribed from #{subscriber_list_slug}"
       else
-        UnsubscribeService.subscription!(active_subscriptions.last, :unsubscribed)
+        UnsubscribeService.unsubscribe!(subscriber, active_subscriptions.last, :unsubscribed)
```

Comment:
Is this passing through just one subscription (via the `.last`)? I'd have thought this only accepted an array?

Reply:
Oh yeah. Turns out none of these tasks are tested 🎉. I've fixed this, but I think adding tests is out of scope for this PR.